src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBScriptBindingProvider.cs

```diff
@@ -74,7 +75,14 @@ public override Type DefaultType
                 {
                     if (Context.Access == FileAccess.Read)
                     {
-                        return typeof(JObject);
+                        if (string.Compare("many", Context.Cardinality, StringComparison.OrdinalIgnoreCase) == 0)
+                        {
+                            return typeof(IEnumerable<JObject>);
```

Comment:
Yeah, only IEnumerable. If I could come up with an easy converter to JArray, I would support it, but all my attempts were failing.